maro/simulator/scenarios/finance/common.py

```diff
@@ -0,0 +1,180 @@
+from enum import Enum
+from typing import Callable
+from functools import partial
+
+
+class FinanceType(Enum):
+    stock = "stock",
+    futures = "futures"
+
+
+class OrderMode(Enum):
+    market_order = "market_order"
+    limit_order = "limit_order"
+    stop_order = "stop_order"
+    stop_limit_order = "stop_limit_order"
+    cancel_order = "cancel_order"
+
+
+class ActionType(Enum):
+    order = "order"
+    cancel_order = "cancel_order"
+    transfer = "transfer"
+
+
+class ActionState(Enum):
+    pending = "pending"
+    success = "success"
+    failed = "failed"
+    expired = "expired"
+    canceled = "canceled"
+
+
+class TradeResult:
+    """Result or a trade order"""
+
+    def __init__(
+            self, sub_engine_name: str, item_index: int, trade_number: int, tick: int,
+            price_per_item: float, tax: float, action_id: int, is_trade_accept: bool = False,
+            is_trade_trigger: bool = True):
+        self.sub_engine_name = sub_engine_name
+        self.item_index = item_index
+        self.trade_number = trade_number
+        self.tick = tick
+        self.price_per_item = price_per_item
+        self.tax = tax
```

Comment:
the tax here is not only the tax, tax + commission ?

Reply:
yes

Reply:
I find there is a ``tax`` field in ``TradeResult`` but no ``commission``. Is the commission included in the tax field of TradeResult? Should be distinguish it.

Reply:
on going

Reply:
done